.github/workflows/pre-commit.yml

```diff
@@ -26,6 +26,8 @@ jobs:
     - uses: actions/checkout@v3
     - uses: ./.github/actions/setup-build
       with:
+        # select a stable GHC version
+        ghc: 9.2.5
```

Comment:
This makes me mad, this should be an error, since it is a required input. But apparently, validation only happens for externally defined actions?  I dunno.